test/unit/reducers/clear_user_data_test.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:crowdleague/actions/auth/clear_user_data.dart';
+import 'package:crowdleague/models/app/app_state.dart';
+import 'package:crowdleague/reducers/app_reducer.dart';
+import 'package:redux/redux.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('ClearUserDataReducer', () {
+    test('Clears any user data from state', () {
```

Comment:
hopefully escaped the capital police 🚓

Reply:
Haha the police have stranger rules than you might think!  The fines are small though 😉 

You have to read b/w the lines a bit, sorry for the lack of the clarity: [Style Guide (WIP) > do-use-a-test-description-that-completes-a-sentence-starting-with](https://github.com/crowdleague/crowdleague/wiki/Style-Guide-(WIP)#do-use-a-test-description-that-completes-a-sentence-starting-with-)

So the `test('...` part should start with lower case... I feel so pedantic! 😞

Reply:
no worries!